### PR TITLE
Updating colour order and uses

### PR DESCRIPTION
**Pull request by kathleenAMA**
So it looks like the greys were disordered and $jet is actually darker than $charcoal. I then got rid of $stormcloud because it's almost the same as $charcoal. @kaytcampbell

### CODE REVIEW

File: lib/ama_layout/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module AmaLayout
-  VERSION = "1.0.0"
+  VERSION = "1.0.1"
```
Comment (rubene) [inline lib/ama_layout/version.rb]: PR #18  have this version, this will have to be bump to 1.0.2?

**Comment by rubene** [comment]
:hankey:

File: lib/ama_layout/version.rb (hunk 1 above)
Comment (kathleenAMA) [inline lib/ama_layout/version.rb]: @rubene I talked to Mike about this on Friday and he said that whoever get's merged first will use this version number and the other PR will have to update.

**Comment by kathleenAMA** [comment]
:wave:

**Comment by rubene** [comment]
:elephant: :elephant: :elephant:

**Comment by kathleenAMA** [comment]
still needs design sign off @kaytcampbell

**Comment by kaytcampbell** [comment]
:art: :art: :art:

**Pull request merged**